public/styles.css

```diff
@@ -1,6 +1,6 @@
 body {
```

Comment:
The changes in this file were done automatically by a tool called Beautify (it might be vs code native, I forget now). There were also some closing-brace issues with your media queries below (due to comments you added) so I fixed those, too.